src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2127,5 +2127,94 @@
   "com.minecolonies.building.largequarry": "Large Quarry",
   "com.minecolonies.coremod.gui.quarry.desc": "To start the Quarry construct a Mine, hire a Quarrier and assign the Quarrier to the Quarry",
   "com.minecolonies.coremod.gui.workerhuts.druid": "Druid",
-  "item.minecolonies.mistletoe": "Mistletoe"
+  "item.minecolonies.mistletoe": "Mistletoe",
+
+  "minecolonies:builder.job.desc": "Constructs buildings and decorations.",
+  "minecolonies:deliveryman.job.desc": "Delivers requested items and blocks to players and colonists.",
+  "minecolonies:miner.job.desc": "Mines out a shaft system under the colony, gathering blocks and ores. Can be a dangerous Job",
+  "minecolonies:quarrier.job.desc": "Requires a Quarry building to mine a large hole. Safe job, but mostly returns common resources.",
+  "minecolonies:lumberjack.job.desc": "Cuts down trees and replants them based on the configured specifications.",
+  "minecolonies:farmer.job.desc": "Farms a large variety of crops at up 5 fields.",
+  "minecolonies:undertaker.job.desc": "Digs up citizens graves, gets their items with a chance to resurrect them.",
```

Comment:
I'm not great at grammar, but I think it would look better as
"Digs up citizens graves to get their items, with a chance to resurrect them."
or 
"Digs up citizens graves and gets their items with a chance to resurrect them."

Reply:
+1 for the first alternative.

Also "citizens'" with an apostrophe after the "s"